src/v/raft/offset_translator.cc

```diff
@@ -75,6 +75,15 @@ bytes serialize_kvstore_key(raft::group_id group, kvstore_key_type key_type) {
 
 } // namespace
 
+bytes offset_translator::kvstore_offsetmap_key(raft::group_id group) {
```

Comment:
i think this is required for the previous commit

Reply:
nice catch